doc/classes/Bone2D.xml

```diff
@@ -1,8 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Bone2D" inherits="Node2D" category="Core" version="3.2">
 	<brief_description>
+		Joint used with [class Skeleton2D] to control and animate other nodes.
 	</brief_description>
 	<description>
+		Use a hierarchy of [code]Bone2D[/code] bound to a [class Skeleton2D] to control, and animate other [class Node2D] nodes.
```

Comment:
I see you got a bit rusty ;) The syntax for class hyperlinks is `[Skeleton2D]` directly, without `class` specifier.

Another thing which we need to reflect in the docs guidelines is that you can now use class hyperlinks to the same class (so here `[Bone2D]`) without worrying about adding too many useless hyperlinks, the docs generator handles those in a cleverer way. @neikeq needed those references to be actual `[Class]` references instead of `[code]Class[/code]` for C# docs.

Reply:
I should've ran makerst. Just fixed it. I kept [code] blocks where there shouldn't be links for other classes, the generated RST looks okay.